awslabs/agent-evaluation
# feat: Add StepfunctionsStack for agent evaluation workflow

*Issue #, if available:*

*Description of changes:*
feat: Add StepfunctionsStack for agent evaluation workflow

- Implement StepfunctionsStack with AWS CDK
- Add Lambda functions for various steps in the workflow:
  - generate_map
  - check_agent_status
  - create_alias
  - delete_alias
  - run_test
  - update_bedrock_agent
- Create Step Functions state machine for agent evaluation process
- Set up EventBridge rule to trigger workflow on S3 object creation
- Configure necessary IAM roles and permissions
- Implement error handling and retry logic in the state machine

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
